src/CONST.js

```diff
@@ -696,6 +696,11 @@ const CONST = {
 
     // There's a limit of 60k characters in Auth - https://github.com/Expensify/Auth/blob/198d59547f71fdee8121325e8bc9241fc9c3236a/auth/lib/Request.h#L28
     MAX_COMMENT_LENGTH: 60000,
+
+    REACT_PDF_PASSWORD_RESPONSES: {
```

Comment:
not a blocker: a comment here would be nice to explain what these values are for